pkgs/top-level/all-packages.nix

```diff
@@ -14197,6 +14197,8 @@ with pkgs;
     inherit (perlPackages.override { pkgs = pkgs // { imagemagick = imagemagickBig;}; }) PerlMagick;
   };
 
+  imagej = callPackage ../applications/graphics/imagej { };
```

Comment:
the following is usually preferred:

```
inherit (callPackage ../applications/graphics/imagej { }) imagej150;
imagej = imagej150;
```

Also I find using sets with a single element premature optimization.